### PR TITLE
Tidy up the readme and make the setup and start scripts more idiomatic.

### PR DESCRIPTION
**Pull request by AWare**
## What does this change?

We're going to be doing some work on this soon, so this is to make setting it up a little easier.

## How can we measure success?

If it runs locally?

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+version: '3'
+services:
+    dyanmo:
+      container_name: dynamo
+      image: amazon/dynamodb-local
+      ports: 
+        - 10005:8000
+      expose: 
+        - 8000
```
Comment (akash1810) [inline docker-compose.yml]: IIUC `expose` is to allow container to container communication. Given we don't have another container, do we need to expose this port atm?

File: migrator/readme.md [hunk 2]
```diff
@@ -0,0 +1,49 @@
+## Running a migration
+
+The migrator sub project produces a executable jar which will migrate data from an R2 database into the path manager.
+
+To run you will need a `migrator.properties` file in the same directory you are executing from. The format of this file is:
```
Comment (akash1810) [inline migrator/readme.md:5]: A few other projects place this file in S3 and then have a `fetch-config.sh` script to bring it down. Is it worth doing similar here?

File: migrator/readme.md [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+
+```
+    databaseAddress=10.0.0.127
+    databaseService=gudevelopersdb.gucode.gnl
+    user=deliveryXX 
+    password=XXXXXXXXXXXX 
+    pathManagerUrl=http://pathmanager.local.dev-gutools.co.uk/
```
Comment (akash1810) [inline migrator/readme.md:12]: nit: this is a dev-nginx address. dev-nginx will setup a redirect from http tp https. This value could be https to remove that hop.

File: readme.md [hunk 4]
```diff
@@ -1,10 +1,6 @@
 Path Manager
 ============
 
-Service that manages paths for a domain. Ensures they are unique etc.
-
-## Rationale
-
```
Comment (mchv) [inline readme.md]: I think you should keep those lines.

File: docker-compose.yml (hunk 1 above)
Comment (AWare) [reply docker-compose.yml]: This was based on the composer one and a brief look at the docs, it can probably go.

File: migrator/readme.md (hunk 2 above)
Comment (AWare) [reply migrator/readme.md:5]: I just lifted the migrator stuff out of the main readme as I didn't think it was something we were likely to use any time soon but didn't want to delete it.

File: migrator/readme.md (hunk 3 above)
Comment (AWare) [reply migrator/readme.md:12]: Could well be, I don't really want to update the migrator docs as this move is a weak deprecation.

File: setup.sh [hunk 5]
```diff
@@ -1,46 +1,37 @@
...
-
-	echo -e "${green}Extracting $dynamoDistFile to $dynamoDir${plain}"
-    tar -C $dynamoDir -xzf $dynamoDistFile
-fi
+fail() {
+    exit_code=$?
+    if [[ "${exit_code}" -ne 0 ]]; then
+        echo -e "\n${red}Setup script failed, please fix errors before starting Composer"
+    else
+        echo -e "\n${green}All done! You can run Composer now."
```
Comment (jennygrahamjones) [inline setup.sh:16]:
```suggestion
        echo -e "\n${red}Setup script failed, please fix errors before starting Path Manager"
    else
        echo -e "\n${green}All done! You can run Path Manager now."
```

File: readme.md [hunk 6]
```diff
@@ -185,97 +184,27 @@ The path manager does not currently support:
...
-    sudo <path_of_dev-nginx>/setup-app.rb <path_of_path_manager>/nginx/nginx-mapping.yml
-    
+The path manager itself is a play app so can be started by the ```run``` command in the `pathManager` sub project in ```sbt```, the app is configured to run
+on port 10000.
     
 The path manager should now be accessible on:
 
-   [https://pathmanager.local.dev-gutools.co.uk/debug](https://pathmanager.local.dev-gutools.co.uk/debug)
+   [https://pathmanager.local.dev-gutools.co.uk/debug](pathmanager.local.dev-gutools.co.uk/debug)
```
Comment (jennygrahamjones) [inline readme.md]: This route doesn't seem to exist.

**Review by jennygrahamjones — APPROVED** [review]

**Pull request merged**